releasenotes/notes/add-pre-commit-hooks-07ce3b82bbe1d7a3.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add playbook to install ``pre-commit`` hooks and register them with git.
```

Comment:
Does pre-commit need to be `codeified`? It seems like just a regular word to me

Reply:
It was only to add emphasis. Though I may use that more than I should.

Will clean up shortly.

Reply:
In this case single backtick was correct, sorry!